Formula/openiked.rb

```diff
@@ -0,0 +1,48 @@
+class Openiked < Formula
+  desc "IKEv2 daemon - portable version of OpenBSD iked"
+  homepage "https://openiked.org"
+  url "https://cdn.openbsd.org/pub/OpenBSD/OpenIKED/openiked-7.2.tar.gz"
+  mirror "https://mirror.edgecast.com/pub/OpenBSD/OpenIKED/openiked-7.2.tar.gz"
+  sha256 "55dc270bc40a121f855d949a25a5ffaeb11e7607e8198ec52160ef54b6946845"
+  license "ISC"
+
+  depends_on "cmake" => :build
+  depends_on "libevent"
+  depends_on "openssl@1.1"  # openssl@3 conflicts with libevent
+
+  on_linux do
+    depends_on "bison"
+  end
+
+  def install
+    system "cmake", ".", "-DHOMEBREW=true",
+                         "-DCMAKE_INSTALL_SYSCONFDIR=#{etc}",
+                         "-DCMAKE_INSTALL_MANDIR=#{man}",
+                         *std_cmake_args
+    system "make"
+    system "make", "install"
```

Comment:
Does this work?
```suggestion
    system "cmake", "-S", ".", "-B", "build",
                         "-DHOMEBREW=true",
                         "-DCMAKE_INSTALL_SYSCONFDIR=#{etc}",
                         "-DCMAKE_INSTALL_MANDIR=#{man}",
                         *std_cmake_args
    system "cmake", "--build", "build"
    system "cmake", "--install", "build"
```

Reply:
in that case I would suggest to solve that like in the openiked-README: https://github.com/openiked/openiked-portable#building-from-source

Reply:
That usage of CMake is a bit outdated, my suggestion is the newer (make independent) way

Reply:
ok, your solution also works